code/modules/surgery/bones.dm

```diff
@@ -30,7 +30,7 @@
 
 //surgery steps
 /datum/surgery_step/glue_bone
-	name = "mend bone"
+	name = "Mend bone"
```

Comment:
You seem to have done this inconsistently with the other steps, `set_bone` for example remains uncapitalised.